tests/scripts/create-dev-cluster.sh

```diff
@@ -63,24 +64,25 @@ show_ceph_dashboard_info() {
     echo "   API_HOST: $PROMETHEUS_API_HOST"
     fi
     echo "==========================="
+    echo " "
+    echo " *** To start using your rook cluster please set the following alias: "
+    echo " "
+    echo "    > " alias kubectl=\"minikube -p $ROOK_PROFILE_NAME kubectl --\"
 }
 
 check_minikube_exists() {
-    minikube profile list > /dev/null 2>&1
-    local retcode=$?
-
-    if [ $retcode -eq 0 ]; then
-        echo "A minikube environment already exists, please use -f to force the cluster creation."
+    if minikube profile list | grep -q $ROOK_PROFILE_NAME; then
+        echo "A minikube profile '$ROOK_PROFILE_NAME' already exists, please use -f to force the cluster creation."
 	exit 1
     fi
 }
 
 setup_minikube_env() {
     minikube_driver="$(get_minikube_driver)"
     echo "Setting up minikube env (using $minikube_driver driver)"
-    minikube delete
-    minikube start --disk-size=40g --extra-disks=3 --driver "$minikube_driver"
-    eval "$(minikube docker-env -p minikube)"
+    minikube delete --profile $ROOK_PROFILE_NAME
```

Comment:
In general, when writing scripts, it's best for other users (anyone reading the script now or in the future, with or without the intent to modify it) to use fully-expressed flag names like `--profile` which you are doing here but not where in the script. This is especially important with scripts that are less often used, like this one. I'd suggest using the long form everywhere.

Reply:
Fixed